src/main.c

```diff
@@ -2583,12 +2586,35 @@ void reset_model() {
     g->time_changed = 1;
 }
 
+void tests(){
```

Comment:
I believe we need doxygen documentation on all tests too. May want to throw that  in just to play it safe.